### PR TITLE
docs: Add VFIO usage example

### PR DESCRIPTION
**Pull request by sameo**
Fixes: #117

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

### CODE REVIEW

File: docs/vfio.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Here we see that our device is on bus 1, slot 0 and function 0 (`01:00.0`).
+
+Now that we have identified the device, we must unbind it from its native driver
+(`rtsx_pci`) and bind it to the VFIO driver instead (`vfio_pci`).
+
+First we add VFIO support to the host:
+
+```
+$ modprobe vfio_pci
```
Comment (sboeuf) [inline docs/vfio.md]: I'd replace with:
```
$ sudo modprobe vfio_pci
```

File: docs/vfio.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Here we see that our device is on bus 1, slot 0 and function 0 (`01:00.0`).
+
+Now that we have identified the device, we must unbind it from its native driver
+(`rtsx_pci`) and bind it to the VFIO driver instead (`vfio_pci`).
+
+First we add VFIO support to the host:
+
+```
```
Comment (sboeuf) [inline docs/vfio.md]:
```
$ sudo modprobe vfio_iommu_type1 allow_unsafe_interrupts
```
is needed in the qemu use case for instance. I'm not sure about the reason though. Might be worth adding it, WDYT?

File: docs/vfio.md (hunk 1 above)
Comment (sameo) [reply docs/vfio.md]: Fixed.

File: docs/vfio.md (hunk 2 above)
Comment (sameo) [reply docs/vfio.md]: Added as well.

**Pull request merged**